ports/directx-dxc/vcpkg.json

```
@@ -0,0 +1,9 @@
+{
+  "name": "directx-dxc",
+  "version-date": "2021-12-08",
+  "description": "DirectX Shader Compiler (LLVM/Clang)",
+  "homepage": "https://github.com/microsoft/DirectXShaderCompiler",
+  "documentation": "https://github.com/microsoft/DirectXShaderCompiler/wiki",
+  "license": null,
+  "supports": "windows & x64 & !uwp & !staticcrt"
```

```suggestion
  "supports": "windows & x64 & !uwp"
```
We might want to also exchange the `x64` clause with `!x86` to support arm64 hosts.

---

It won't work on arm64 hosts running Windows 10 on ARM. It will only work on arm64 hosts running Windows 11. I'll be updating the port in a few weeks anyhow when the upstream source provides an explicit arm64 binary.